.phan/config.php

```diff
@@ -14,6 +14,7 @@
 	]
 );
 
+// We intentionally explicitly specified this so that it picks up on deprecations, etc..
```

Comment:
Maybe the wording could be changed to this for better usage in this context?
> // We explicitly specify this so that it
> // picks up on deprecations, and other issues

Reply:
Thank you, I worried the syntax of the sentence I mixed is valid or not.

```suggestion
// We explicitly specify this so that it
// picks up on deprecations, and other issues
```